ape_safe/accounts.py

```diff
@@ -109,11 +131,14 @@ def fallback_handler(self) -> Optional[ContractInstance]:
             return None
 
     @cached_property
-    def client(self) -> SafeClient:
+    def client(self) -> BaseSafeClient:
         chain_id = self.provider.chain_id
         if chain_id not in self.account_file["deployed_chain_ids"]:
             raise ClientUnavailable(f"Safe client not valid on chain '{chain_id}'.")
 
+        if self.provider.network.name == "local":
```

Comment:
if you import it from the top:

```python
from ape.api.networks import LOCAL_NETWORK_NAME
```

```suggestion
        if self.provider.network.name == LOCAL_NEWORK_NAME:
```